*** tests/A-009-009-M-ok.out	Wed Oct 11 15:17:36 2023
--- tests/A-009-009-M-ok.outhyp	Wed Oct 11 16:57:37 2023
***************
*** 5,15 ****
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Menu Principal
  1 - Novo
  2 - Abrir
  3 - Guardar
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
  Escolha uma opção: 
\ No newline at end of file
--- 5,16 ----
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: Menu Principal
  1 - Novo
  2 - Abrir
  3 - Guardar
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: 
\ No newline at end of file
