*** tests/A-010-010-M-ok.out	Wed Oct 11 15:17:36 2023
--- tests/A-010-010-M-ok.outhyp	Wed Oct 11 16:57:37 2023
***************
*** 1,8 ****
  Menu Principal
  1 - Novo
  2 - Abrir
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Menu Principal
  1 - Novo
  2 - Abrir
  3 - Guardar
--- 1,12 ----
  Menu Principal
  1 - Novo
  2 - Abrir
+ 3 - Guardar
+ 4 - Menu de Edição
+ 5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Abrir: Operação inválida: Problema ao abrir ficheiro: Erro a processar ficheiro saved09
! Menu Principal
  1 - Novo
  2 - Abrir
  3 - Guardar
***************
*** 18,25 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 3;2|5
! Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
--- 22,28 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
***************
*** 28,35 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 3;1|10
! Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
--- 31,38 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Número inválido!
! Escolha uma opção: Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
***************
*** 38,43 ****
--- 41,47 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Menu Principal
  1 - Novo
  2 - Abrir
